plugins/GSdx/GSLocalMemory.cpp

```diff
@@ -41,7 +41,7 @@
 		for(int x = _r.left; x < _r.right; x += w >> 3) \
 		{ \
 			const uint8* src = BlockPtr(_base + off->block.col[x]); \
-			uint8* dst = &_dst[x * bpp]; \
+			dst = &_dst[x * bpp]; \
```

Comment:
will it? All that was happening (from what I can see) is it was hiding the previous declaration, but presumably it's changing dst and returning a bool to say if it did or not?